packages/backend/app/Services/Telegram.ts

```diff
@@ -0,0 +1,94 @@
+import axios from 'axios'
+import { DateTime } from 'luxon'
+
+import util from 'util'
+import { IMsgToTelegram } from './../../common/interfaces/IMsgToTelegram'
+import Logger from '@ioc:Adonis/Core/Logger'
+import { IPositiveResponse, INegativeResponse } from '../../common/interfaces/IResponse'
+import { ITelegram } from '../../common/interfaces/Itelegram'
+import telegramConfiguration from 'Config/telegram'
+import { creatingErrMsg, creatingOkMsg } from '../../common/helpers/creatingResponse'
+
+export default class TelegramBot {
+  private readonly config: ITelegram
+  constructor () {
+    this.config = telegramConfiguration
+  }
+
+  protected formatDate = (date: DateTime): string => {
+    return date
+      .setLocale('ru')
+      .setZone('Europe/Moscow')
+      .toLocaleString(DateTime.DATETIME_MED)
+  }
+
+  private async sendRequest (
+    path: string,
+    _data?: IMsgToTelegram
+  ): Promise<{ ok: boolean } & IMsgToTelegram> {
+    const response = await axios.get(
+      `https://api.telegram.org/bot${this.config.telegram_bot_token}${path}`
+    )
+    return response.data
+  }
+
+  public async sendMsgToTelegram (
+    id: number,
+    apartmentAddress: string,
+    dateFrom: DateTime,
+    dateTo: DateTime,
+    fixedTotalPrice: number,
+    name: string,
+    phone: string,
+    paymentUrl: string
+  ): Promise<INegativeResponse | IPositiveResponse<IMsgToTelegram>> {
+    const msg = this.createMessageBody(
+      id,
+      apartmentAddress,
+      dateFrom,
+      dateTo,
+      fixedTotalPrice,
+      name,
+      phone,
+      paymentUrl
+    )
+
+    try {
+      const telegramResponse = await this.sendRequest(
+        `/sendMessage?&chat_id=${this.config.telegram_chat_id}&parse_mode=html&text=${msg}`
+      )
+      return creatingOkMsg(telegramResponse)
```

Comment:
```suggestion
      return creatingOkMsg(telegramResponse)
```
Реальный ответ слать на фронт не стоит. Просто текстом, что успешно забронировал. И вообще лучше, чтобы успешный ответ уходил, когда получилось забронировать. А неуспешная отправка в ТГ только в логах отражалась.